.travis.yml

```diff
@@ -14,18 +14,25 @@ env:
   # we want to test the most recent few releases
   - V=0.5.3
   - V=0.5.4
+  - V=0.6.0rc2
```

Comment:
Adding this seemed like the best way to avoid possibly regressing in future since the 0.5.x's don't have this behavior.